refinery/core/api.py

```diff
@@ -2023,6 +2023,17 @@ def obj_create(self, bundle, **kwargs):
         new_ext_group.manager_group.user_set.add(user)
         return bundle
 
+    def obj_delete(self, bundle, **kwargs):
+        user = bundle.request.user
+        ext_group = super(ExtendedGroupResource, self).obj_get(
```

Comment:
Don't believe the `super` call is necessary here, could be confusing for someone new to look at

try:
` ext_group = self.obj_get(bundle, **kwargs)`